### ISSUE TITLE
Remapping Controls

### ISSUE THREAD

**Issue by Pcitek** [ClusterM/retroarch-clover#142]
I've remapped A to B and B to Y for NES, GBC, and GBA games but when I play SNES games that use Retroarch, it also uses the same control mappings. Is there anyway to keep the controls on default for Retroarch SNES games while NES, GBC, and GBA games keep my intended configurations?

**Comment by doratos** [comment]
+1
We ask for different RetroArch configuration(i mean general, not core one) for each system emulated.
There are system like N64 that you can't use analog controls with snes controller by changing input core configuration, but you need to change RetroArch general input to use them. But in this way you will change also the input configuration of all the other system emulated by RetroArch.

**Comment by KMFDManic** [comment]
RetroArch settings, Quick Settings, Save Core Override

**Comment by markched** [comment]
By doing this you'll save the mapping for that one specific system?

**Comment by KMFDManic** [comment]
@markched, correct.  But, it does not work for "everything" and, or all Cores.  But, it does the trick for most:)  I will try to do a video/tutorial on alternate ways to pull off this, as well.

**Comment by markched** [comment]
So we lock in the core, do I need to go back into other cores and change the controller settings back to default?
